extensions/virtuallistview/src/VirtualListView.tsx

```diff
@@ -345,6 +345,15 @@ export class VirtualListView<ItemInfo extends VirtualListViewItemInfo>
             if (!(item.key in newItemMap)) {
                 // If we're deleting an item that's above the current render block,
                 // update the adjustment so we avoid an unnecessary scroll.
+                
+                // Update focused item if it's the one removed, if we're unable to, reset focus
+                if (item.key === this.state.lastFocusedItemKey) {
+                    if (!this._selectSubsequentItem(FocusDirection.Down, false) &&
```

Comment:
Both conditions are equivalent

```typescript
this._selectSubsequentItem(FocusDirection.Down, false) && !this._selectSubsequentItem(FocusDirection.Down, false)
```

I think it is a typo. ;)

Reply:
Good catch